src/towncrier/newsfragments/349.enhancement

```diff
@@ -0,0 +1 @@
+Changed type of `versiondata.date` in the main template from `string` to `date`. This allows users to customize the date rendering in the template with `strftime`, like so: `{{versiondata.date.strftime("%B")}}`.
```

Comment:
Minor comment.

I think is best when this text is written in the style of a public release note and not as a commit message :)

```suggestion
The `versiondata.date` is not `date` instance. This allows users to customize the date rendering in the template with `strftime`, like so: `{{versiondata.date.strftime("%B")}}`.
```